src/python/WMComponent/JobArchiver/JobArchiverPoller.py

```diff
@@ -69,8 +69,12 @@ def __init__(self, config):
             logging.exception(msg)
             raise JobArchiverPollerException(msg)
 
+        self.handleWorkflowInjection = getattr(self.config.JobArchiver,
```

Comment:
Instead of defining another parameter, how about we use the same logic as this:
https://github.com/dmwm/WMCore/blob/master/src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py#L58
?